scripts/karma.coffee

```diff
@@ -10,7 +10,7 @@
 
 module.exports = (robot) ->
 
-  robot.hear /^@?(.*?)(\+\+|--).*?$/, (response) ->
+  robot.hear /@(.*?)(\+\+|--).*?$/, (response) ->
```

Comment:
can we just eliminate spaces between name and `--`?
I'd be in favor of:
`/@([^ ]+?)(\+\+|--).*?$/`
...for inline matches. That way we don't trip it with dashes -- I typically use this pattern all the time!

Reply:
Interesting idea, I always do the space, however.

Reply:
Also that regex, doesn't like to match anything...
![image](https://user-images.githubusercontent.com/28048/113048558-4b3f5480-9168-11eb-8295-8d66d92b3d9e.png)